docs/source/usage/workflows/ionization.rst

```diff
@@ -0,0 +1,187 @@
+
+    .. code-block:: c++
+
+        namespace picongpu
+        {
+            #ifndef PARAM_IONIZATIONCURRENT
+            #    define PARAM_IONIZATIONCURRENT None
+            #endif
+
+            /* Example Nitrogen */
+            using ParticleFlagsNitrogen = MakeSeq_t<
+                particlePusher<UsedParticlePusher>,
+                shape<UsedParticleShape>,
+                interpolation<UsedField2Particle>,
+                current<UsedParticleCurrentSolver>,
+                atomicNumbers<ionization::atomicNumbers::Nitrogen>                                                 // <-- from step 1
+                effectiveNuclearCharge<ionization::effectiveNuclearCharge::Nitrogen_t>, // <-- from step 1
+                ionizationEnergies<ionization::energies::AU::Nitrogen_t >               // <-- from step 2
+                massRatio<MassRatioNitrogen>,                                           // <-- from step 3
+                chargeRatio<ChargeRatioNitrogen>,                                       // <-- from step 3
+                ionizers<MakeSeq_t<               // <-- comma separated list of all ionizers of this species with
+                                                  // species of electrons macro particles to be created upon ionization
+                    particles::ionization::BSIEffectiveZ<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ADKLinPol<Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT>,
+                    particles::ionization::ThomasFermi<Electrons>>>>;
+
+            using ParticleAttributesNitrogen = MakeSeq_t<position<position_pic>, momentum, weighting, boundElectrons>;
+            using Nitrogen = Particles<PMACC_CSTRING("N"), ParticleFlagsNitrogen, ParticleAttributesNitrogen>;
+        } // namespace picongpu
+
+    .. note::
+        Remember that you can define multiple electron species and associate them with your different ion species to track the ionisation processes separately!
+
+6. Create ion macro particles in the simulation and initialize them in :code:`speciesInitialization.param`.
+
+    .. code-block:: c++
+
+        namespace picongpu::particles
+        {
+            /* create nitrogen charge state 1 and electrons to neutralize the simulation */
+            using InitPipeline = pmacc::mp_list<
+                CreateDensity< ... some densityProfile... , startPosition::Random, Nitrogen>,
+                ManipulateDerive<manipulators::binary::DensityWeighting, Nitrogen, Electrons>,
+                Manipulate<manipulators::unary::ChargeState<1u>;, Nitrogen>>;
+        } // namespace picongpu::particles
+
+    .. attention::
+        Remember to initialize the simulation charge neutral to avoid creating ghost background charge densities.
```

Comment:
This is not really a problem. For many simulations, you work with the ion "ghost" background. How is the following sentence:
```suggestion
        Ensure the simulation is initialized as charge neutral whenever the respective species should move, to avoid creating unintended ghost background charge densities.
```

Reply:
done